cql3/expr/expression.hh

```diff
@@ -815,15 +815,30 @@ bool has_only_eq_binops(const expression&);
 
 /// Required for fmt::join() to work on expression.
 template <>
-struct fmt::formatter<cql3::expr::expression> {
+class fmt::formatter<cql3::expr::expression> {
+    bool _debug = true;
+public:
     constexpr auto parse(format_parse_context& ctx) {
-        return ctx.end();
+        auto it = ctx.begin();
+        if (ctx.begin() != ctx.end()) {
+            switch (*it) {
+            case 'd':
+                _debug = true;
+                ++it;
+                break;
+            case 'u':
+                _debug = false;
+                ++it;
+                break;
+            }
```

Comment:
I think it would be good to add a comment what `u` and `d` mean. As I understand this isn't any standard convention, just something that was programmed in the formatter. Maybe also next to `operator<<`? That would be the first place for me to look if I was interested in printing expressions.

Reply:
I wanted to have :user and :debug. I'll try to do that, a comment here won't help someone looking at a format() line.